## Description: Specify an address family in http.c:get_port()
## Origin/Author: Adam Glasgall <adam@crossproduct.net>
Index: gnome-user-share-3.0.0/src/http.c
===================================================================
--- gnome-user-share-3.0.0.orig/src/http.c	2011-09-22 15:00:09.086505068 -0400
+++ gnome-user-share-3.0.0/src/http.c	2011-09-22 15:00:00.766504944 -0400
@@ -77,6 +77,7 @@
 	memset (&addr, 0, sizeof (addr));
 	addr.sin_port = 0;
 	addr.sin_addr.s_addr = INADDR_ANY;
+	addr.sin_family = AF_INET;
 
 	reuse = 1;
 	setsockopt (sock, SOL_SOCKET, SO_REUSEADDR, &reuse, sizeof (reuse));
